### PR TITLE
add RGBA mode

### PR DESCRIPTION
**Pull request by zvezdochiot**
add RGBA mode (RGB*).

### CODE REVIEW

File: src/render.c [hunk 1]
```diff
@@ -101,17 +101,14 @@ void render(gint32 image_ID,
                 double ay0 = 0.0;
                 double ay1 = 0.0;
 
-                gdouble bc[3 * 3 * 3];  // 3x3 neighborhood, with RGB
+                gdouble bc[3 * 3 * channels];  // 3x3 neighborhood, with RGB
```
Comment (pannacotta98) [inline src/render.c:104]: Comment is no longer accurate

**Review by pannacotta98 — CHANGES_REQUESTED** [review]
👍

File: src/render.c (hunk 1 above)
Comment (zvezdochiot) [reply src/render.c:104]: Hi @pannacotta98 .

Maybe `// 3x3 neighborhood`

Comment (pannacotta98) [reply src/render.c:104]: Sounds fine to me :)

Comment (zvezdochiot) [reply src/render.c]: Hi @pannacotta98 .
```c
                gdouble bc[3 * 3 * channels];  // 3x3 neighborhood
```
:heavy_check_mark:

**Pull request merged**